contracts/lib/reality-eth/BalanceHolder.sol

```diff
@@ -2,7 +2,7 @@
 
 pragma solidity ^0.8.20;
 
-import "../interfaces/IBalanceHolder.sol";
+import "../../interfaces/IBalanceHolder.sol";
```

Comment:
One balance holder is in mixin and the other one in libs?

If parts of this is already audited, then I would love if they are imported from a gitsubmodule, just as we do it with the polygon code

Reply:
I think forge will yell at me if we try to reference the reality.eth code from `@reality.eth/contracts` as is because it sets a different solc version. So mostly what I figured I could do was put the reality.eth contracts that we use functionally unchanged in their own directory, then at the end work out whether we have to maintain our own version or go ahead and upgrade `@reality.eth/contracts`. There are also some places where we'd benefit from some reality.eth changes, for example #83, so it may well be worth doing, but I'd like to get a bit further along before deciding.

BalanceHolder.sol is particularly weird because it's used as part of reality.eth that makes sense, but then it's randomly used in some other part of our code, which sort-of makes sense from a code reuse pov but it's a bit weird since the job of reality.eth isn't to provide a BalanceHolder.

Reply:
OK so I put both the BalanceHolders in the same place for consistency. [4fc33d0](https://github.com/RealityETH/subjectivocracy/pull/88/commits/4fc33d022f2e0fba38533ab041474fd303d013ae)

Made issue #90 for potentially replacing the whole thing with the`@reality.eth/contracts` npm package later on.